client/bindings/java/iota-client-java/lib/src/test/java/org/iota/IndexerApiTest.java

```diff
@@ -78,6 +83,7 @@ public void testGetFoundryOutputIdByFoundryId() throws ClientException {
     }
 
     @Test
+    @Disabled
```

Comment:
Why are we disabling them ?

Reply:
We do a full query on the indexer which isn't good. We need to find an other solution (in case we really want to test what these tests are testing - in a previous review we thought that these tests are not really helpful anyways). Until we didn't find a better solution we should deactivate this for the CI IMO. I would create an issue for that to not forget, would this be ok ?

Reply:
(I didn't find a better solution yet to rewrite these tests unfortunately)